scripts/build-rpm.sh

```diff
@@ -6,13 +6,17 @@ set -o pipefail
 
 source "$(dirname "$0")/common.sh"
 
-# There is no build step, so we can omit the tarball
-# step that is used in the sdw config rpm
+# Default to prod (rpm_spec directory) but
+# allow override for test RPM purposes
+SPECFILE_DIR=${SPECFILE_DIR:-rpm_spec}
 
+# %_projdir is a custom macro that lets us
+# refer to static files inside the project directory
 rpmbuild \
     --quiet \
-    --define "_topdir $PWD/rpm-build" \
-    -bb --clean "rpm-build/SPECS/${PROJECT}.spec"
+    --define "_projdir ${PWD}" \
```

Comment:
(this is the custom macro so we can refer to static files in the project directory during packaging, instead of building a tarball/source)

Reply:
Yeah, we need to target/support qubes builder. So based on https://github.com/QubesOS/qubes-issues/issues/9312#issuecomment-2182863415 we do need to build from a tarball and need to re-introduce that step unfortunately :/

Semi-relatedly, is it straightforward to set up qubes builder in CI?